### ISSUE TITLE
Cloudfare Captcha Verification error

### ISSUE THREAD

**Issue by ramakant7153** [ranjan-mohanty/vfs-appointment-bot#89]
Dear Ranjan,

I am facing captcha Verification error as it is asking for verify you are human every time.

**Comment by okflow** [comment]
if any body can expert in making bot i have some project for that devloper

whatsapp- +60 13-908 8506

**Comment by ramakant7153** [comment]
I am not expert but can help you and also enhance my skills

**Comment by aavashn** [comment]
i need a vfs bot, that can book multiple appointments to vfs global, that can work in realtime, please who is expert here , please come inbox me +351920496902, only geniune person please

**Comment by iamyegor** [comment]
@aavashn Do you still need this bot? I can help you

**Comment by ramakant7153** [comment]
Yes

**Comment by armogit** [comment]
@iamyegor I also would like some help to get this working.

**Comment by loc0w** [comment]
anyone who needs a bot can contact telegram @whoisturki.

**Comment by aavashn** [comment]
> [@aavashn](https://github.com/aavashn) Do you still need this bot? I can help you



> [@aavashn](https://github.com/aavashn) Do you still need this bot? I can help you

**Comment by aavashn** [comment]
Yes, i still want, i need your contact, telegram or WhatsApp please

**Comment by aavashn** [comment]
> anyone who needs a bot can contact telegram @whoisturki.

**Comment by aavashn** [comment]
Yes i want, but no scam huh, all payments will be done only after successful testing and only if it works

**Comment by loc0w** [comment]
telegram id @whoisturki

> Yes i want, but no scam huh, all payments will be done only after successful testing and only if it works

**Comment by aavashn** [comment]
Yes i already texted you

**Comment by muhammad1928** [comment]
> > [@iamyegor ](https://github.com/aavashn) Do you still need this bot? I can help you
> 
> are you still willing to help someone in need :) it happens to be i am stuck with this automation for 3 weeks now. if you are able to help i would appreciate it. thanks in advance

**Comment by jbpf1** [comment]
I need vfs bot Guine Bissau to Portugal

**Comment by aavashn** [comment]
yes I need it
